src/main/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryController.java

```diff
@@ -680,15 +690,24 @@ public AdvisoryUpdateResponse changeComment(
             @Parameter(
                     in = ParameterIn.PATH,
                     description = "The ID of the comment to change."
-            ) long commentId,
+            ) String commentId,
             @RequestParam @Parameter(description = "Optimistic locking revision.") String revision,
             @RequestBody String newCommentText
```

Comment:
The RequestBody should be better commented

Reply:
added more documentation